cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -59,10 +59,10 @@ var hostfolderAddCmd = &cobra.Command{
 
 func init() {
 	HostfolderCmd.AddCommand(hostfolderAddCmd)
-	hostfolderAddCmd.Flags().BoolVarP(&instanceOnly, "instance-only", "", false, "Define host folder only for the current cluster instance.")
+	hostfolderAddCmd.Flags().BoolVarP(&instanceOnly, "instance-only", "", false, "Defines the host folder only for the current cluster.")
 
 	// Windows-only
 	if runtime.GOOS == "windows" {
-		hostfolderAddCmd.Flags().BoolVarP(&usersShare, "users-share", "", false, "Define host folder for the Users share on a Windows host.")
+		hostfolderAddCmd.Flags().BoolVarP(&usersShare, "users-share", "", false, "Defines the shared `Users` folder as the host folder on a Windows host.")
```

Comment:
The single quotes are for the parsed docs, not for being copied/pasted along with the folder name. This way when we generate the command reference the folder name will show up in the correct formatting.